### ISSUE TITLE
Zurücknehmen von geteilten Keys?

### ISSUE THREAD

**Issue by eltomato89** [Deazl/pr0keys#65]
Ich meine das wäre mal möglich gewesen. Gibts aktuell leider nicht.
Habe leider den falschen Key geteilt, der jetzt im System ist (Aber nicht mehr gültig).

Habe dafür jetzt natürlich rep bekommen.

Das einstellen von falschen Keys ist ein Grundsatzproblem ...

**Comment by xlaum** [comment]
heißt du zufällig "nice"?

**Comment by eltomato89** [comment]
Nein,  aber von ihm habe ich auch schon einen Key bekommen,  der nicht funktioniert hat.  Ist dir das auch passiert?Dann scheint er das häufiger zu machen...                                                                                                                                                                                                                   Von:notifications@github.comGesendet:4. März 2016 2:26 nachm.An:pr0keys@noreply.github.comAntworten:reply@reply.github.comCc:eltomato@googlemail.comBetreff:Re: [pr0keys] Zurücknehmen von geteilten Keys? (#65)  heißt du zufällig "nice"?

—Reply to this email directly or view it on GitHub.

**Comment by xlaum** [comment]
https://gyazo.com/8f615c073d80fe8bde04751f4c66647b

**Comment by Deazl** [comment]
https://github.com/Deazl/pr0keys/issues/57#issuecomment-192331277
